### PR TITLE
Add Civic Data Lab India spider

### LINKED ISSUE #494: Civic Data Lab India
**Links to bulk download and/or API documentation**
https://github.com/CivicDataLab/himachal-pradesh-health-procurement-OCDS

**Links to bulk download and/or API endpoints**
https://github.com/CivicDataLab/himachal-pradesh-health-procurement-OCDS/tree/master/ocds-health-procurement-data

**Link to CRM issue**
https://crm.open-contracting.org/issues/6252

**Priority**
anytime

Excel files

### PR DESCRIPTION
**Pull request by aguilerapy**
closes #494

### CODE REVIEW

**Comment by yolile** [comment]
@aguilerapy could you run `scrapy updatedocs`?  This command should be run every time we add a new spider https://kingfisher-collect.readthedocs.io/en/latest/contributing/index.html#commit-the-spider

File: kingfisher_scrapy/spiders/india_civic_data_lab.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+class IndiaCivicDataLab(SimpleSpider):
+    """
+    Domain
+      Civic Data Lab India
+    Bulk download documentation
+      https://github.com/CivicDataLab/himachal-pradesh-health-procurement-OCDS/
+    """
+    name = 'india_civic_data_lab'
+    data_type = 'release_list'
```
Comment (yolile) [inline kingfisher_scrapy/spiders/india_civic_data_lab.py:15]: Hmm, this should be release_package, shouldn't it? Same for bolivia_agetic actually

**Review by yolile — CHANGES_REQUESTED** [review]

File: kingfisher_scrapy/spiders/india_civic_data_lab.py (hunk 1 above)
Comment (aguilerapy) [reply kingfisher_scrapy/spiders/india_civic_data_lab.py]: Done for Bolivia and India.

**Comment by aguilerapy** [comment]
I added the possibility to give more arguments to `unfflaten` with the `unflatten_args` class attribute.

File: kingfisher_scrapy/spiders/india_civic_data_lab.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    def start_requests(self):
+        url = 'https://github.com/CivicDataLab/himachal-pradesh-health-procurement-OCDS'
+        yield scrapy.Request(url, meta={'file_name': 'list.html'}, callback=self.parse_list)
+
+    @handle_http_error
+    def parse_list(self, response):
+        for path in response.xpath('//div[@role="rowheader"]/span/a/@href').getall():
+            if path.endswith('.xlsx'):
+                yield scrapy.Request(
```
Comment (yolile) [inline kingfisher_scrapy/spiders/india_civic_data_lab.py:30]: can we just:

```python
                yield self.build_request(
                    f'https://github.com{path}?raw=true',
                    formatter=components(-1)
                )
```
?
same for Bolivia actually. 

and then in https://github.com/open-contracting/kingfisher-collect/blob/2380f96531630aa203c7db9a188a5d2d3d6a1f50/kingfisher_scrapy/base_spider.py#L190

```python       
 if not file_name.endswith(('.json', '.zip', '.xlsx', '.csv')):
````

**Review by yolile — CHANGES_REQUESTED** [review]

File: kingfisher_scrapy/spiders/india_civic_data_lab.py (hunk 2 above)
Comment (aguilerapy) [reply kingfisher_scrapy/spiders/india_civic_data_lab.py]: Done.

**Review by yolile — APPROVED** [review]

**Pull request merged**